tests/api_test_helper.js

```diff
@@ -0,0 +1,52 @@
+const RECIPE_API = "/api/recipe";
+const USER_API = "/api/users";
+const AUTH_API = "/api/auth/login";
+const ADD_RECIPE_API = "/api/recipe/add";
+const STATUS_CODE = require("../shared/errorCode");
+
+const supertest = require("supertest");
+const app = require("../app");
+const api = supertest(app);
+
+const createUserAndLogin = async (user) => {
+  await api.post(USER_API).send(user);
+  return api.post(AUTH_API).send(user).expect(STATUS_CODE.OK);
+};
+
+const addRecipe = async (recipe, authHeader) => {
+  return api
+    .post(ADD_RECIPE_API)
+    .set("Authorization", authHeader)
+    .send(recipe)
+    .expect(STATUS_CODE.CREATED)
+    .expect("Content-Type", /application\/json/);
+};
+
+const updateRecipe = (recipeId, recipe, token) => {
+  return api
+    .patch(`${RECIPE_API}/${recipeId}/edit`)
+    .set("Authorization", `Bearer ${token}`)
+    .send(recipe);
+};
+
+const getProperty = async (endpoint, id, property) => {
+  const response = await api.get(`/${endpoint}/${id}`);
+  return response.body[property];
+};
```

Comment:
Seems weird to call api when your method is called `getProperty`. Ideally your method should do one thing.